content/en/serverless/step_functions/redrive.md

```diff
@@ -24,6 +24,17 @@ To take action on a Step Function in Datadog:
 4. Click on the **Failed** pill to open a redrive modal.
 5. Click the **Redrive** button.
 
+## Tracing redrives
+When monitoring redriven executions, the Waterfall view is recommended as the large gap between the original execution and redrive can make the Flame Graph view imperceptible.
```

Comment:
```suggestion
When monitoring redriven executions, use the Waterfall view, as the large gap between the original execution and redrive can make the Flame Graph view imperceptible.
```
This is a stronger recommendation but in this context I think it's okay -- just trying to reduce the sentence length here.